src/components/media/SearchResults.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement } from 'react';
+import * as ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+
+// Logic
+import { seasonYear } from '../../logic/dateFunctions';
+
+// GraphQL
+import { SearchMediaByTitleQuery } from './search_media.types';
+
+// Components
+import Title from '../../styles/components/Title';
+import { SubtypeTag } from '../../styles/components/Tag';
+import Search from 'src/styles/layouts/Search';
+import Poster from 'src/styles/components/Poster';
+
+interface Props {
+  data: SearchMediaByTitleQuery;
+}
```

Comment:
we don't need an interface for this.

Reply:
Same as above, I was using a snippet and wasn't really thinking

Reply:
I couldn't remove the interface for that for some reason. It just threw errors at me everywhere.

Reply:
left an example below as a suggest change